ui/ts/models/log.ts

```diff
@@ -46,11 +54,14 @@ module Models {
       });
 
       private _url(): string {
-        let url: string = "/_status/local/log";
-        if (this.level() != null) {
-          url += "/" + this.level();
+        let url: string = "/_status/logs/";
+        if (this.node() != null) {
+          url += this.node();
         }
         url += "?";
+        if (this.level() != null) {
+          url += "level=" + this.level() + "&";
```

Comment:
jQuery has this feature :troll:

On Wed, Jul 22, 2015 at 1:00 PM, Bram Gruneir <notifications@github.com>
wrote:

> In ui/ts/models/log.ts
> <https://github.com/cockroachdb/cockroach/pull/1762#discussion_r35236378>:
>
> >          }
> >          url += "?";
> > +        if (this.level() != null) {
> > +          url += "level=" + this.level() + "&";
>
> @mrtracy <https://github.com/mrtracy>: sadly, no, you would figure this
> would be something built into js
> @tamird <https://github.com/tamird>: I was already using
> encodeURIComponent for the pattern, but I extended this to each component
> just in case.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/1762/files#r35236378>.
>